WordPress/gutenberg
# Update block inserter and library terminology and labels

Fixes https://github.com/WordPress/gutenberg/issues/61483

## What?

The term 'block inserter' is used inconsistently in user-facing strings. All text and label referencing it can be improved.

## Why?

- WordPress features should always use title case capitalization.
- The verb 'Toggle' should be avoided.
- The term 'Block Library' is used as synonym of 'Block Inserter' but that's less than ideal in the UI when controls refer to the same feature but use different terminology.
- All X 'Close' icon button can be labeled just 'Close' when the context already provides the 'what' is going to be closed.

## How?

- Makes the term 'Block Inserter' always use title case capitalization.
- Removes the verb 'Toggle' from the Block Inserter toggle button.
- Replaces the term 'Block Library' with 'Block Inserter' everywhere it makes sense.
- Simplifies the label of the Block Inserter X close button from 'Close block inserter' to 'Close'.
- Adjusts tests accordingly.

## Testing Instructions




- Go to the Post editor.
- Hover of focus the button that opens the Block Inserter.
- Observe the button tooltip text is 'Block Inserter' title cased and the verb 'Toggle' has been removed.
- Open the Block Inserter.
- Inspect the source in your browser dev tools and observe the element with CSS class `interface-navigable-region interface-interface-skeleton__secondary-sidebar` which is the ARIA landmark with role=region is now labeled `Block Inserter` instead of `Block Library`.
- Hover or focus the X close button at the top right of the Block Inserter and observe the tooltip text is now 'Close'.
- Search through the entire codebase and observe all the occurrences of 'Block Inserter' are now title case, except in the changelog files.

### Testing Instructions for Keyboard


## Screenshots or screencast
